.github/workflows/native-tests.yml

```diff
@@ -0,0 +1,24 @@
+name: GraalVM native tests
+on:
+  schedule:
+    # daily at 4:00 UTC
+    - cron: '0 4 * * *'
+  workflow_dispatch:
```

Comment:
let's see how long these tests take to run, it may make sense to move them into `build.yml` or `build-common.yml`